metrics_handler/metrics_test.py

```diff
@@ -165,6 +165,18 @@ def test_within_bounds(self, value, bounds, inclusive, expected):
     within_bounds = metrics.within_bounds(value, *bounds, inclusive)
     self.assertIs(within_bounds, expected)
 
+  # To use this test, replace the args to compute_memory_metrics with your
```

Comment:
If this can only be run manually, I suggest moving it to a different file/script.

Reply:
good idea, created a new file `manual_tests.py` for these and similar unit tests